nx.json

```diff
@@ -11,7 +11,8 @@
           "test:format",
           "test:build",
           "build"
-        ]
+        ],
+        "accessToken": "OTgzYjM0YmUtNzlmNi00MWFhLWIyZmEtODA0NTgwODNiZjY4fHJlYWQtd3JpdGU="
```

Comment:
is that access token something we should provide as an env variable instead?

Reply:
Yes, I think what we normally recommend is creating a "public" key that only allows read access to include here.

Then, we'd create a private key (or possible a set of private keys) with read-write access. We'd set this in our github secrets and use it in CI, and optionally trusted maintainers could add a private key (either shared or one unique to them) and set it via an environment variable.

I'll confirm with the cloud team the recommendation for open-source repos tomorrow.